src/components/table/table-header-filters/FilterCheckbox.tsx

```
@@ -16,6 +17,7 @@ export default function FilterCheckbox({
   countryCode,
   customName,
 }: {
+  setRowSelection: React.Dispatch<React.SetStateAction<{}>>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)